src/schemas/validation/schema.yaml

```diff
@@ -733,6 +733,9 @@ $defs:
             type: boolean
         $ref: '#/$defs/examples'
     $ref: '#/$defs/specification-extensions'
```

Comment:
This line seems to be implied by the second allOf entry.

Reply:
@ralfhandl thanks, fixed!